['Traceback (most recent call last):\n  File "/var/folders/x3/l43tt5dd3y19dmn0cq8rhm7r0000gq/T/tmp05pa_llh/solution_0.py", line 39, in <module>\n    print(Solution().maxTotalReward(rewardValues = [1,1,3,3]))\nTypeError: maxTotalReward() got an unexpected keyword argument \'rewardValues\'\n', 'Traceback (most recent call last):\n  File "/var/folders/x3/l43tt5dd3y19dmn0cq8rhm7r0000gq/T/tmp05pa_llh/solution_1.py", line 39, in <module>\n    print(Solution().maxTotalReward(rewardValues = [1,6,4,3,2]))\nTypeError: maxTotalReward() got an unexpected keyword argument \'rewardValues\'\n']